lua/wikis/commons/GetMatchGroupCopyPaste.lua

```diff
@@ -228,6 +228,28 @@ function CopyPaste.singleMatch(frame, args)
 	return CopyPaste._generateCopyPaste(display)
 end
 
+---@param frame Frame
+---@param args table
+---@return Html
+function CopyPaste.matchPage(frame, args)
+	if not args then
+		args = Arguments.getArgs(frame)
+	end
+
+	args.generateMatchPage = true
+
+	local bestof = tonumber(args.bestof) or 3
+	local opponents = tonumber(args.opponents) or 2
+	local mode = WikiSpecific.getMode(args.mode)
+
+	local display = WikiSpecific.getMatchCode(bestof, mode, 1, opponents, args)
```

Comment:
would we possibly want to assert the existence of that function, so that one gets a better error message if a wiki tries to set up the form without adding the needed module changes?

Reply:
ah true